StarWars5e.Models/Enums/MonsterSize.cs

```diff
@@ -8,6 +8,7 @@ public enum MonsterSize
         Medium,
         Large,
         Huge,
-        Gargantuan
+        Gargantuan,
+        Garagantuan, //TODO: compensates for spelling error in rulebook. Remove when fixed
```

Comment:
Should keep tabs on this and remove when typo is fixed.

Reply:
Going to hold on merging for a day, Gali usually has a pretty quick turnaround on typos, I bet he fixes this and some of the others by tomorrow morning

Reply:
Typo fixed, so I removed this.